content/v2.0/write-data/sample-data/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Sample data
+description: >
+  Use sample data to familiarize yourself with time series data and InfluxDB.
+  InfluxData provides many sample time series datasets to use with InfluxDB
+  and InfluxDB Cloud.
+menu:
+  v2_0:
+    parent: Write data
+weight: 105
+---
+
+Use sample data to familiarize yourself with time series data and InfluxDB.
+InfluxData provides many sample time series datasets to use with **InfluxDB** and **InfluxDB Cloud**.
+
+{{< children >}}
+
```

Comment:
To explore and learn time series and InfluxDB, use:
- Sample data sets:
    - Air sensor....
    - Bird...
- (InfluxDB Cloud only) [Demo data buckets](...)